package-lock.json

```diff
@@ -10031,7 +10031,7 @@
             "integrity": "sha512-m62ShEObQ39CfralilEQRjH6oAMtNCV1xJyEx5LpRYUVN+EviphDgUc/F3hnYbADmkiNs67Y+3ylmlG7Lnu+FA==",
             "dev": true,
             "requires": {
-                "es5-ext": "0.10.53",
+                "es5-ext": "^0.10.50",
```

Comment:
Can you remove the `package-lock.json` changes?

Reply:
yup done!